DOCKER.md

```diff
@@ -0,0 +1,123 @@
+# Development with Docker
+
+## Features
+
+* Run development server requirements such as MySQL and Redis in containers.
+
+## Installing Docker
+
+Our Docker development environment requires at least Docker Compose version 2.23 or higher.
+These instructions will walk you through installing a compatible version.
+
+To install Docker, see the appropriate section below:
+[macOS](#macos), [Ubuntu](#ubuntu), [Windows](#windows)
+or consult your package management for your OS.
+
+### macOS
+
+1. Install Docker itself:
+```shell
+brew install docker
+```
+
+1. Install Docker Compose:
+    1. ```shell
+       export DOCKER_CONFIG=${DOCKER_CONFIG:-$HOME/.docker}
+       mkdir -p $DOCKER_CONFIG/cli-plugins
+       curl -SL https://github.com/docker/compose/releases/download/v2.27.0/docker-compose-darwin-$(uname -m) -o $DOCKER_CONFIG/cli-plugins/docker-compose
```

Comment:
I'm on an M1 mac and this didn't work, `uname -m` returns arm64 and the releases are suffixed `aarch64`, which the internet tells me is effectively the same thing? ([release link for this version](https://github.com/docker/compose/releases/download/v2.27.0/docker-compose-darwin-aarch64))

Reply:
grrr that this happens. I guess it needs an extra instruction for arm-based Macs.